### PR TITLE
Fixed tests

### PR DESCRIPTION
**Pull request by szemek**
# Fixes assertion checking Dockerfile, removes assertion for docker-compose.yml

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation about the functionality in the appropriate .md file
- [ ] I have added in line documentation to the code I modified

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-ruby?pullRequest=281) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-ruby?pullRequest=281) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-ruby?pullRequest=281) it.</sub>

File: .codeclimate.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ engines:
   rubocop:
     enabled: true
   bundler-audit:
-    enabled: true
+    enabled: false
```
Comment (thinkingserious) [inline .codeclimate.yml]: According to this: https://codeclimate.com/github/sendgrid/sendgrid-ruby/pull/281

It seems the problem is that we don't have a Gemfile.lock file. It seems like that would be the correct fix.

Thoughts?

Comment (szemek) [reply .codeclimate.yml]: @thinkingserious As far as I know it is not recommended to keep Gemfile.lock in repository when developing gem
https://github.com/sendgrid/sendgrid-ruby/blob/master/.gitignore#L34-L36
http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/

As you can see in [sendgrid-ruby.gemspec](https://github.com/sendgrid/sendgrid-ruby/blob/master/sendgrid-ruby.gemspec) only gems constraints are provided, so every time when somebody installs sendgrid-ruby gem, she/he gets the versions of dependencies suitable to context of installation (whether it's done by `bundle install` or `gem install` different set of versions might be installed).

I'm not sure if it makes sense to perform bundler audit check.

Comment (thinkingserious) [reply .codeclimate.yml]: Thanks @szemek! I appreciate the quick response.

That makes sense.

Comment (szemek) [reply .codeclimate.yml]: @thinkingserious do you think my changes can be merged, do I need to improve anything or should I close my PR?

Comment (thinkingserious) [reply .codeclimate.yml]: @szemek,

Yes, I'm just waiting for this issue to hit the top of my backlog. Thanks!

**Pull request closed without merging**